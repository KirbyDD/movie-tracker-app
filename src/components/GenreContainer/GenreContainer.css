.genre-container {
  display: grid;
  grid-template-columns: repeat(auto-fill,minmax(220px, 1fr));
  background-image: linear-gradient(rgba(14, 14, 14, 0.609), #2c2b2b34);
  height: calc(100vh - 50px);
}