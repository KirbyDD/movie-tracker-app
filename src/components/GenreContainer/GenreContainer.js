import React from 'react';

const GenreContainer = () => {
  return (
    null
  )
}

export default GenreContainer;