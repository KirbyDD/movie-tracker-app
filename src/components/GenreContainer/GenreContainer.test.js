import { mapStateToProps, GenreContainer } from './GenreContainer';
import React from 'react';
import { shallow } from 'enzyme';

describe('GenreContainer', () => {
  describe('GenreContainer Component', () => {
    let wrapper, instance;
    beforeEach(() => {
      const mockMovies = [{title: 'Ryans day', poster_path: 'mockUrl', id: 1}]
      wrapper = shallow(<GenreContainer movies={mockMovies}/>)
      instance = wrapper.instance()
    })

    it('Should match the Snapshot', () => {
      expect(wrapper).toMatchSnapshot()
    })

    it('should include movies in the props', () => {
      console.log(wrapper.props())

    })
  })
})