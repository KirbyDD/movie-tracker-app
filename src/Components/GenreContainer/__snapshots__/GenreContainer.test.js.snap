// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenreContainer GenreContainer Component Should match the Snapshot 1`] = `
<section
  className="genre-container"
>
  <MovieSnippet
    id={1}
    key="1"
    path="mockUrl"
  />
</section>
`;
